account_wallet/models/account_wallet_type.py

```diff
@@ -0,0 +1,94 @@
+# © 2015  Laetitia Gangloff, Acsone SA/NV (http://www.acsone.eu)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
```

Comment:
Maybe we can do that in a ROADMAP.

Reply:
I tried to implement that point, but is in fact not possible, because we can not copy items that are fields like (account_id) that : 
- are required
- should be uniq by constraint.